src/test/ui/fn/dyn-fn-alignment.rs

```diff
@@ -1,6 +1,5 @@
 // run-pass
 
-#![feature(unsized_locals)]
```

Comment:
how does this compile without the new feature gate?

Reply:
Because the feature gate is turned on in liballoc https://github.com/rust-lang/rust/pull/72029/files#diff-aac3a6e0b1cf9ed92cf8215c2d29f733R120 and the compiler uses it for https://github.com/rust-lang/rust/blob/c1e079a4ede241fde71ed5773e29ba5ddce98d61/src/liballoc/boxed.rs#L1030-L1036. So it's nothing that the user activates.

Reply:
O_o so this PR is an instant stabilization of this feature?

Reply:
It is the same we had with unsized_locals https://github.com/rust-lang/rust/pull/72029/files#diff-aac3a6e0b1cf9ed92cf8215c2d29f733R121

Reply:
Well seemingly it is not the same as so far the test here still requires a feature gate?

Reply:
@RalfJung unsure exactly what you meant, the feature gate was added by mistake. It was never required, if right now I go on master and remove this line everything works. As I've said this is turned on in liballoc to be used in `impl FnOnce for Box`.

Reply:
> if right now I go on master and remove this line everything works.

Ah, that is the part which I missed.